eNMS/forms/__init__.py

```diff
@@ -46,8 +48,23 @@ def __new__(cls, name, bases, attrs):
                     "integer": IntegerField,
                     "json": JsonField,
                     "string": StringField,
+                    "select": SelectField,
+                    "multiselect": SelectMultipleField,
                 }[values.get("type", "string")]
             form_kw = {"default": values["default"]} if "default" in values else {}
+            if field in [SelectField, SelectMultipleField]:
+                if "choices" in values:
```

Comment:
Why do we need that ? Is there a use-case for having a select/multiselect with no choice i.e empty ?

Reply:
There probably is not  - that's a good point.  I'll remove this.